templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="s">
+    <li>
+        <a href="#">Home page</a>
+    </li>
+    <li>
+        <a href="#">Manufacturers</a>
+    </li>
+    <li>
+        <a href="#">Cars</a>
+    </li>
+    <link>
+        <a href="#">Drivers</a>
+    </li>
```

Comment:
There is a mistake in the HTML structure: the `<link>` tag is incorrectly used here. It should be a `<li>` tag to properly close the list item. Replace `<link>` with `<li>`.